lighthouse-core/report/report-generator.js

```diff
@@ -132,6 +132,7 @@ class ReportGenerator {
       // XSS, define a renderer that only transforms links and code snippets.
       // All other markdown ad HTML is ignored.
       const renderer = new marked.Renderer();
+      renderer.br = () => '<br>';
```

Comment:
Yeah just not sure how to appropriately target the `pre` tag then for margin only when its following other text. Also I can't get the `pre` to generate with triple ticks, it still does `code` for me after cherry-picking :/